css/style.css

```diff
@@ -601,88 +664,101 @@ div.left-arrow{
     position: relative;
 }
 
-div.field{
+div.field {
     display: flex;
 }
 
-div.formComponent div input#NameInput, div.formComponent select{
+div.formComponent div input#NameInput, div.formComponent select {
     left: 15px;
     position: relative;
 }
 
-input#NameInput{ position: sticky !important; }
+input#NameInput {
+    position: sticky !important;
+}
 
-div#formName, div#formRadios, div#formMajor, div#formGrad{
+div#formName, div#formRadios, div#formMajor, div#formGrad {
     display: inline-flex;
     justify-content: center;
 }
 
-form#newMember{
+form#newMember {
     display: flex;
     flex-direction: column;
     gap: 15px;
     top: 20px;
     position: relative;
 }
 
-#newMemberSubmit{
+#newMemberSubmit {
     width: 35%;
     margin: auto;
 }
 
 
 /* Start CSS for codeOfConduct */
-#codeOfConduct .header{
+#codeOfConduct .header {
     text-align: center;
 }
 
-#codeOfConduct p.header{ margin-bottom: calc(2.75vw + 18px); }
+#codeOfConduct p.header {
+    margin-bottom: calc(2.75vw + 18px);
+}
+
+#codeOfConduct p {
+    font-size: calc(1.1vw + 9px);
+}
+
+#codeOfConduct h1 {
+    font-size: calc(1.6vw + 18px);
+}
 
-#codeOfConduct p{ font-size: calc(1.1vw + 9px); }
-#codeOfConduct h1{ font-size: calc(1.6vw + 18px); }
-#codeOfConduct li{ font-size: calc(1vw + 8px); }
+#codeOfConduct li {
+    font-size: calc(1vw + 8px);
+}
 
-#iAgree{
+#iAgree {
     color: black;
     background-color: papayawhip;
     font-size: calc(1.2vw + 12px);
     border: calc(.1vw + 2px) solid black;
     border-radius: calc(.8vw + 4px);
-    padding: calc(.4vw + 2px) calc(.6vw + 3px) ;
+    padding: calc(.4vw + 2px) calc(.6vw + 3px);
     width: calc(7ch + 1vw + 8px);
     text-decoration: none;
     text-align: center;
     margin: auto;
     margin-top: calc(.8vw + 4px);
     position: absolute;
     left: 0;
-    right: 0;    
+    right: 0;
 }
 
-#iAgree:hover{
+#iAgree:hover {
     background-color: #fdd89b;
 }
+
 /* End CSS for codeOfConduct */
 
 
-a.cornerButton, button.cornerButton {/* Navigation buttons in top corners */
+a.cornerButton, button.cornerButton {
+    /* Navigation buttons in top corners */
     color: black;
     border: 1px solid black;
     padding: 4px;
     text-decoration: none;
-    position: absolute;
```

Comment:
I believe `z-index: 1` is a slightly better solution as it avoids content shifting that would otherwise be caused by `position: static`.